pom.xml

```diff
@@ -155,6 +155,16 @@
       <scope>test</scope>
       <version>${rest-assured.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.jmolecules</groupId>
+      <artifactId>jmolecules-ddd</artifactId>
+      <version>1.9.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jmolecules</groupId>
+      <artifactId>jmolecules-layered-architecture</artifactId>
+      <version>1.9.0</version>
```

Comment:
I would create the property to avoid 1.9.0 repetition and be able to updated easily